src/content/blog/maintaining-ddev-tests-contributor-training.md

```diff
@@ -24,61 +23,61 @@ Here's our September 18, 2024 [Contributor Training](/blog/category/training) on
 DDEV has three main types of tests:
 
 - Go “pkg” tests, that don’t use an external `ddev` binary. These are mostly a "pure" form of Go test. There are both end-to-end tests and some unit tests.
-- Go “cmd” tests that rely on a matching external `ddev` binary. These are almost scripted tests, written in Go, but calling out to execute the `ddev` binary and exercise it. 
+- Go “cmd” tests that rely on a matching external `ddev` binary. These are almost scripted tests, written in Go, but calling out to execute the `ddev` binary and exercise it.
 - Container tests written using bats (bash).
 
-Golang tests are in files named `*_test.go` and the tests themselves are functions named `Test*`.  For example the test [TestConfigFunctionality](https://github.com/ddev/ddev/blob/507cfca2508b97786b80e8b8c83ea17f5c0fea20/pkg/ddevapp/config_test.go#L1458) is in the file [config_test.go](https://github.com/ddev/ddev/blob/master/pkg/ddevapp/config_test.go).
+Golang tests are in files named `*_test.go` and the tests themselves are functions named `Test*`. For example the test [TestConfigFunctionality](https://github.com/ddev/ddev/blob/507cfca2508b97786b80e8b8c83ea17f5c0fea20/pkg/ddevapp/config_test.go#L1458) is in the file [config_test.go](https://github.com/ddev/ddev/blob/master/pkg/ddevapp/config_test.go).
 
 ## How many tests are there? How many lines of test code?
 
-According to cloc there are currently 22,322 lines of go code: (`cloc ./pkg ./cmd --include-lang=Go --not-match-f='.*_test\.go$'`) and 14,853 lines of go test code (*_test.go): (`cloc ./pkg ./cmd --include-lang=Go --match-f='.*_test\.go$'`)
+According to cloc there are currently 22,322 lines of go code: (`cloc ./pkg ./cmd --include-lang=Go --not-match-f='.*_test\.go$'`) and 14,853 lines of go test code (_\_test.go): (`cloc ./pkg ./cmd --include-lang=Go --match-f='._\_test\.go$'`)
```

Comment:
```suggestion
According to cloc there are currently 22,322 lines of go code: (`cloc ./pkg ./cmd --include-lang=Go --not-match-f='.*_test\.go$'`) and 14,853 lines of go test code (`*_test.go`): (`cloc ./pkg ./cmd --include-lang=Go --match-f='.*_test\.go$'`)
```

Reply:
Hmm ... assuming the original change was made by Prettier, we might need to a `prettier-ignore-start/end` block around the code to prevent Prettier from "correcting" it again.

Reply:
Confirmed.  I added the suggestion locally and Prettier was still happy. Thanks @stasadev

Reply:
@stasadev I committed your suggested change, thanks! 👍  So that is resolved / fine now, correct?

https://pr-format-prettier.ddev-com-front-end.pages.dev/blog/maintaining-ddev-tests-contributor-training/

Reply:
Yes, all good.